src/guide_restli_server.md

```diff
@@ -1755,6 +1767,10 @@ Valid parameter types and return types for action are:
     example, `StringMap`, `LongMap`, and so on.
 -   Custom types
 
+To use Arrays and Maps of non-primitive type (Eg record, enum) as return type,
```

Comment:
nits:
- "To use Arrays and Maps of non-primitive **types**"
- "as return **types**"
- "should be defined as **a data model**"
- As Junchuan pointed out, "e.g." or "E.g." is the standard spelling